site/app/Resources/views/template/email_template.html.twig

```diff
@@ -0,0 +1,249 @@
+              <tr>
+                <td>
+                  <!-- COPY -->
+                  <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                    <tr>
+                      <td align="left" style="font-size: 25px; font-family: Helvetica, Arial, sans-serif; color: rgba(63, 176, 172, 1); padding-top: 30px;" class="padding-copy">Hi {{ username }},</td>
+                    </tr>
+                    <tr>
+                      <td align="left" style="font-size:14px;padding: 20px 0 5px 0;border-bottom: 2px solid #e2e2e2">You have a new Notification from <b>LibreCores</b>.</td>
+                    </tr>
+                    <tr>
+                      <td align="justify" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;" class="padding-copy">
+                        <p>{{ content }}</p>
+                      </td>
+                    </tr>
+                  </table>
+                </td>
+              </tr>
+            </table>
+          </td>
+        </tr>
+      </table>
+    </td>
+  </tr>
+</table>
+
+<!-- ONE COLUMN W/ BOTTOM IMAGE SECTION -->
+<table border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;">
+  <tr>
+    <td bgcolor="#f8f8f8" align="center" style="padding: 70px 15px 70px 15px;" class="section-padding-bottom-image">
+      <table border="0" cellpadding="0" cellspacing="0" width="500" class="responsive-table">
+        <tr>
+          <td>
+            <table width="100%" border="0" cellspacing="0" cellpadding="0">
+              <tr>
+                <td>
+                  <!-- COPY -->
+                  <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                    <tr>
+                      {# TODO:Update the link after merge of notification settings page #}
+                      <td align="justify" style="padding: 10px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;" class="padding-copy">
+                        You are recieving this notification because you are subscribed to email notifications from LibreCores by default.
+                        You can snooze these notifications if you want to. For detailed preferences,
+                        go to <a href="#">notification settings</a> of your LibreCores account.
+                      </td>
+                    </tr>
+                  </table>
+                </td>
+              </tr>
+            </table>
+          </td>
+        </tr>
+      </table>
+    </td>
+  </tr>
+</table>
+
+<!-- FOOTER -->
+<table border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;">
+  <tr>
+    <td bgcolor="#ffffff" align="center">
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" align="center">
+        <tr>
+          <td style="padding: 20px 0px 20px 0px;">
+            <!-- UNSUBSCRIBE COPY -->
```

Comment:
You need to include a way to unsubscribe from these emails. Include a link to the settings page.

Reply:
Sure